src/terminal/adapter/ut_adapter/adapterTest.cpp

```diff
@@ -794,8 +812,9 @@ class TestGetSet final : public ConGetSet
         _fExpectedCursorVisible = _fCursorVisible;
 
         // Attribute default is gray on black.
-        _wAttribute = FOREGROUND_BLUE | FOREGROUND_GREEN | FOREGROUND_RED;
-        _wExpectedAttribute = _wAttribute;
+        _attribute = TextAttribute();

```

Comment:
nit: constructor call copy assignment seems unusual/uncommon; is there a more pleasant way to achieve this?

Reply:
Not sure it's any better (`= {}`), but maybe more idiomatic?